scripts/split_eafs.py

```diff
@@ -92,11 +92,9 @@ def read_eaf(ie):
     if 'PARTICIPANT' in params:
         speaker_id = params['PARTICIPANT']
 
-    annotations_data = []
     i = 0
     for ann in annotations:
         skip = False
-        ref_annotation = []
         start = ann[0]
         end = ann[1]
         annotation = ann[2]
```

Comment:
There are a few things to note here, firstly instead of creating a looping variable such as `i` and incrementing it each loop prefer to use [enumerate](https://docs.python.org/3/library/functions.html#enumerate):

```python
for i, ann in enumerate(annotations):
```
This gets rid of the need to define a looping variable and removes potential bugs from managing that.

Going further I'd unpack values directly here:
```python
for i, (start, end, annotation) in annotations:
```
This fills in the variables for you and will immediately raise errors from unpacking if annotations data isn't exactly length 3. This can help prevent some subtle bugs.